pkg/chartverifier/pyxis/pyxis.go

```diff
@@ -114,33 +119,51 @@ func IsImageInRegistry(repository string, version string, registry string) (bool
 			json.Unmarshal(body, &registriesBody)
 
 			if len(registriesBody.PyxisRegistries) > 0 {
+
 				var tags []string
+				var shas []string
 				found = false
 				for _, reg := range registriesBody.PyxisRegistries {
-					for _, repo := range reg.Repositories {
-						if repo.Repository == repository && repo.Registry == registry {
-							for _, tag := range repo.Tags {
-								if tag.Name == version {
-									found = true
-									break
+					if len(imageSha) > 0 {
+						if strings.Compare(imageSha, reg.ParsedData.Digest) == 0 {
```

Comment:
From the docs: https://golang.org/pkg/strings/#Compare
```
Compare is included only for symmetry with package bytes.
It is usually clearer and always faster to use the built-in string
comparison operators ==, <, >, and so on.
```

Reply:
OK and done.